powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspect.java

```diff
@@ -74,6 +77,13 @@ public final class LambdaLoggingAspect {
         }
 
         LEVEL_AT_INITIALISATION = LOG.getLevel();
+
+        String logEvent = System.getenv("POWERTOOLS_LOGGER_LOG_EVENT");
+        if (logEvent != null) {
+            LOG_EVENT = Boolean.parseBoolean(logEvent);
```

Comment:
can we try catch if the `parseBoolean` fails and log a warn, to avoid an exception ?

Reply:
We could with some effort, but I think this is before logger is initialized, though